workers/unity/Assets/Gdk/Core/Error/ForwardingDispatcher.cs

```diff
@@ -0,0 +1,67 @@
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class ForwardingDispatcher : ILogDispatcher
+    {
+        private Connection connection;
+
+#if UNITY_EDITOR
+        private readonly LoggingDispatcher loggingDispatcher = new LoggingDispatcher();
+#endif
+
+        public void SetConnection(Connection connection)
+        {
+            this.connection = connection;
+        }
+
+        public void HandleLog(LogType type, LogEvent logEvent)
+        {
+#if UNITY_EDITOR
+            if (Application.isEditor)
+            {
+                loggingDispatcher.HandleLog(type, logEvent);
+            }
+#endif
+
+            if (connection == null)
+            {
+                return;
```

Comment:
If you are disconnected but still running you still want to log locally. Most likely use case is running in the editor.